lib/urbanopt/reporting/default_reports/reporting_period.rb

```diff
@@ -57,14 +57,16 @@ class ReportingPeriod
                       :net_site_energy_kwh, :net_source_energy_kwh, :total_utility_cost_dollar, :net_utility_cost_dollar, :utility_costs_dollar, :electricity_kwh, :natural_gas_kwh, :propane_kwh, :fuel_oil_kwh, :other_fuels_kwh, :district_cooling_kwh,
                       :district_heating_kwh, :water_qbft, :electricity_produced_kwh, :end_uses, :energy_production_kwh, :photovoltaic,
                       :fuel_type, :total_cost_dollar, :usage_cost_dollar, :demand_cost_dollar, :comfort_result, :time_setpoint_not_met_during_occupied_cooling,
-                      :time_setpoint_not_met_during_occupied_heating, :time_setpoint_not_met_during_occupied_hours, :hours_out_of_comfort_bounds_PMV, :hours_out_of_comfort_bounds_PPD #:nodoc:
+                      :time_setpoint_not_met_during_occupied_heating, :time_setpoint_not_met_during_occupied_hours, :hours_out_of_comfort_bounds_PMV, :hours_out_of_comfort_bounds_PPD,
+                      :emissions_kg, :future_annual_emissions_kg, :future_hourly_emissions_kg, :historical_annual_emissions_kg, :historical_hourly_emissions_kg  #:nodoc:
```

Comment:
other projects (BuildingSync, SEED, BPS efforts in general) are standardizing on this unit for reporting GHG emissions: MtCO2e (metric ton CO2e)

Should we also do that, or keep it in kg?

Reply:
we can change it later if needed.